.gitignore

```diff
@@ -3,3 +3,4 @@
 
 ios/RCTWKWebView.xcodeproj/xcuserdata/*
 ios/RCTWKWebView.xcodeproj/project.xcworkspace/xcuserdata/*
+macos/RCTWKWebView.xcodeproj/xcuserdata
```

Comment:
I'd maybe take this out in case it removes build schemes - https://stackoverflow.com/a/13954760

Reply:
I agree that this change should maybe be in a separate PR, but I do think the user data should be excluded from version control.

It's an important point that's raised, but the solution should be to make the necessary schemes shared schemes, rather than keeping the user data under version control.